grammars/javascript.cson

```diff
@@ -1360,6 +1360,28 @@
         ]
       }
       {
+        'begin': '(?<=innerHTML\\s*(\\+?=)\\s*)(`)'
```

Comment:
Lookbehinds only support fixed-width regexes, so `\\s*` and stuff won't work.  I would suggest going back to the previous regex and trying to make that one work.